### DISCUSSION TITLE
Get Module Properties_ Mapping for Status Word

### DISCUSSION THREAD

**Discussion by SathishDi** [dmroeder/pylogix#248] [Q&A]
I'm requesting the module properties of a CompactLogix controller

![Code](https://github.com/dmroeder/pylogix/assets/114287930/ca33c64e-97ab-4f39-bce8-6ac40c124737)
![Output](https://github.com/dmroeder/pylogix/assets/114287930/f9b16d91-5c34-4af7-8bbc-c6fdfe486f71)

I am getting the value of 12384 for Status and 65 for State. Someone please help to understand the result. Is there any mapping details for the Status and State?

**Comment by TheFern2** [comment]
I've never payed much attention to module properties a quick test:

```
(venv) PS C:\Users\fernandob\git\pylogix-playground> python .\main.py
12384 49  <- Run mode
(venv) PS C:\Users\fernandob\git\pylogix-playground> python .\main.py
12400 49  <- Program mode
(venv) PS C:\Users\fernandob\git\pylogix-playground> python .\main.py
45152 49  <- Test mode
```

I am not too sure what any of these numbers mean though.
